File: POO JAVA/Aula12/src/aula12/Peixe.java
package aula12;

public class Peixe extends Animal{
    
    //Atributos da Classe
    private String corEscama;
    
    //Metodo Acessor e Modificador
    public String getCorEscama() {
        return corEscama;
    }
    public void setCorEscama(String corEscama) {
        this.corEscama = corEscama;
    }
    
    //Metodo da Classe
    public void soltarBolha(){
        System.out.println("Soltando bolhas na agua ...");
    }

    //Metodos de Sobreposição da Classe Aminal
    @Override
    public void locomover() {
        System.out.println("Nadando ...");
    }
    @Override
    public void alimentar() {
        System.out.println("Comando substancia na agua ...");
    }
    @Override
    public void emitirSom() {
        System.out.println("Peixe nao emite som !!!!");
    }    
    
}

File: POO PHP/Aula12/Peixe.php
<?php

require_once 'Animal.php';

class Peixe extends Animal{

    //Atributo da Classe
    private $corEscama;
    
    //Metodo Acessor e Modificador
    public function getCorEscama() {
        return $this->corEscama;
    }
    public function setCorEscama($corEscama): void {
        $this->corEscama = $corEscama;
    }

    //Metodos Sobreposto da Classe Animal
    public function alimentar() {
        echo "<p>Comendo substancias na agua ...</p>";
    }
    public function emitirSom() {
        echo "<p>Peixe nao emiti som !!!</p>";
    }
    public function locomover() {
        echo "<p>Nadando muito ...</p>";
    }

    //Metodo da Classe
    public function soltandoBolha() {
        echo "<p>Soltando bolhas na agua ...</p>";
    }
}
